var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -69,7 +72,9 @@ def cmake_args(self):
             '-DBUILD_SHARED_LIBS=%s' % ('+shared' in self.spec)]
 
         if self.spec.variants['build_type'].value == 'Debug':
-            cmake_cxx_flags.append('-DDEBUG_REALM', '-DDEBUG_LEGION', '-ggdb')
+            cmake_cxx_flags.append('-DDEBUG_REALM')
+            cmake_cxx_flags.append('-DDEBUG_LEGION')
+            cmake_cxx_flags.append('-ggdb')
```

Comment:
You can use extend to add a list of flags

Reply:
Like 
```
cmake_cxx_flags.extend('-DDEBUG_REALM')
cmake_cxx_flags.extend('-DDEBUG_LEGION')
cmake_cxx_flags.extend('-ggdb')
```
?

Reply:
Like:
```python
cmake_cxx_flags.extend([
    '-DDEBUG_REALM',
    '-DDEBUG_LEGION',
    '-ggdb',
])
```

Reply:
Updated.